Gruntfile.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(grunt) {
+  grunt.initConfig({
+    jshint: {
+      all: ['Gruntfile.js', 'index.js']
```

Comment:
@Anirudh94 OK then we should probably switch header -> header.jsx, and lint all *.js files. There's also this: https://github.com/STRML/JSXHint.